source/vscode/src/qirGeneration.ts

```diff
@@ -167,15 +184,13 @@ async function getQirForProgram(
     );
     clearTimeout(compilerTimeout);
   } catch (e: any) {
-    log.error("Codegen error. ", e.toString());
     if (e.toString() === "terminated") {
       throw new QirGenerationError(
         "QIR generation was cancelled or timed out.",
       );
     } else {
       throw new QirGenerationError(
-        `QIR generation failed due to error: "${e.toString()}". Please ensure the code is compatible with a QIR profile ` +
-          "by setting the target QIR profile to 'base' or 'adaptive_ri' and fixing any errors.",
+        `QIR generation failed due to error: "${e.toString()}". Please ensure the code is compatible with the selected QIR profile.`,
```

Comment:
Does this look user-friendly in the UI or do want to further refine this messaging?

Reply:
updated error messages